File: src/test/alg2/program.js
//Using a for loop
var items = [2, 5, 3, 7, 8, 10, 15, 18, 24, 111, 12, 19, 87];

function itemSearch(items, item) {
    for(var i=0; i < items.length; i++) {
        if(items[i] === item) {
            console.log("Found item " + item + " at index " + i);
            return true;
        }
    }
    //item not found 
    return false;
}

var item = itemSearch(items, 15);
if(!item) {
    console.log('Item does not exist!');
}

File: src/test/alg2/program.cs
using System;
using System.Collections.Generic;

namespace dotnet
{
    class Program
    {
        static void Main(string[] args)
        {
            List<int> items = new List<int>() { 2, 5, 3, 7, 8, 10, 15, 18, 24, 111, 12, 19, 87 };
            bool item = itemSearch(items, 15);
            if(!item) {
                Console.WriteLine("Item does not exist!");
            }
        }

        static bool itemSearch(List<int> items, int item){
            for (int i = 0; i < items.Count; i++)
            {
                if(items[i] == item){
                    Console.WriteLine("Found item " + item + " at index " + i);
                    return true;
                }
            }
            return false;
        }
    }
}
